source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -433,7 +433,7 @@ namespace AdaptiveCardsSharedModelUnitTest
 
         TEST_METHOD(LinkBasicValidationTest_ValidLinkTestWithEscapedDelimiters)
         {
-            MarkDownParser parser("[[cool link!]](https://contoso.com/New%20Document%20(1\\).docx)");
```

Comment:
inner parenthesis is correctly handled after [PR](https://github.com/microsoft/AdaptiveCards/pull/6922) and is no longer needed to escape it.